src/root.mk

```diff
@@ -1,31 +1,55 @@
-# You can change the pdf viewer to your preferred
-# one by commenting every line beginning by
-# `PDFVIEWER' except the one with your pdf viewer
-#PDFVIEWER=evince # GNOME
-#PDFVIEWER=okular # KDE
-#PDFVIEWER=xpdf # lightweight
-PDFVIEWER=xdg-open # Default pdf viewer - GNU/Linux
-#PDFVIEWER=open # Default pdf viewer - Mac OS
 ifneq (,$(filter $(TYPE),exam test))
   MAIN_NAME=${NAME}-${OPTION}${CODE}-${TYPE}-${YEAR}-${MONTH}-${MINMAJ}
 else
   MAIN_NAME=${NAME}-${OPTION}${CODE}-${TYPE}
 endif
 
-ALL+=$(MAIN_NAME).pdf
+ifeq ($(strip ${TYPE}),summary)
+	ifdef NUM
+		OUT_MAIN_NAME:=Synthèse${NUM}-${NAME}-${OPTION}${CODE}
+	else
+		OUT_MAIN_NAME:=Synthèse-${NAME}-${OPTION}${CODE}
+	endif
+else ifeq ($(strip ${TYPE}),notes)
+	OUT_MAIN_NAME:=CM-${NAME}-${OPTION}${CODE}
+else ifneq (,$(filter $(TYPE),exercises))
+	ifdef NUM
+		OUT_MAIN_NAME:=APE${NUM}-${NAME}-${OPTION}${CODE}
+	else
+		OUT_MAIN_NAME:=APE-${NAME}-${OPTION}${CODE}
+	endif
+	OUT_MAIN_NAME:=APE-${NAME}-${OPTION}${CODE}
+else ifeq ($(strip ${TYPE}),mcq)
+	OUT_MAIN_NAME:=QCM-${NAME}-${OPTION}${CODE}
+else ifeq ($(strip ${TYPE}),errata)
+	OUT_MAIN_NAME:=Books-${NAME}-${OPTION}${CODE}
+else ifeq ($(strip ${TYPE}),test)
+	OUT_MAIN_NAME:=${OPTION}${CODE}-${YEAR}-${MONTH}
+else ifeq ($(strip ${TYPE}),formulaire)
+	ifdef NUM
+		OUT_MAIN_NAME:=formulaire${NUM}-${NAME}-${OPTION}${CODE}
+	else
+		OUT_MAIN_NAME:=formulaire-${NAME}-${OPTION}${CODE}
+	endif
+else ifeq ($(strip ${TYPE}),exam)
+	OUT_MAIN_NAME:=${OPTION}${CODE}-${YEAR}-${MONTH}
+	ifneq (,$(filter $(strip ${MINMAJ}),Maj Majeure)) 
+		OUT_MAIN_NAME:=$(strip $(OUT_MAIN_NAME))-Majeure
+	else ifneq (,$(filter $(strip ${MINMAJ}),Min Mineure))
+		OUT_MAIN_NAME:=$(strip $(OUT_MAIN_NAME))-Mineure
+	endif
+	ifneq (,$(filter $(strip ${NAME}),meca elec))
+		OUT_MAIN_NAME:=$(strip $(OUT_MAIN_NAME))-${NAME}
+	endif
+endif
 
-MAIN_NAME_SOL=${MAIN_NAME}-Sol
+INPUT:=$(strip ${QUADRI} ${NAME} ${OPTION} ${CODE} ${TYPE} ${YEAR} ${MONTH})
 
-ifneq (,$(filter $(TYPE),exam test exercises mcq))
-  ALL+=$(MAIN_NAME_SOL).pdf
-  FULL=${MAIN_NAME_SOL}.pdf
-else
-  FULL=${MAIN_NAME}.pdf
-endif
+DIRPATH:=/home/martin/Documents/Onedrive/EPL-Drive
```

Comment:
This won't work on someone else computer. Maybe read these variables from a separate file so that someone can change it without the need to change `root.mk`

Reply:
Yes :) . The only thing that remains to be done in order to compiler it is to change the output directory in line 2 of config.yml :wink: .